packages/storybook/src/components-react/form-field-label/FormFieldLabel.mdx

```diff
@@ -0,0 +1,55 @@
+import { Canvas, Controls, Description, Markdown, Meta } from "@storybook/blocks";
+import markdown from "@utrecht/form-label-css/README.md?raw";
+import * as FormFieldLabelStories from "./FormFieldLabel.stories";
+import { CitationDocumentation } from "../../utils/CitationDocumentation";
+
+<Meta of={FormFieldLabelStories} />
+
+# Lux Form Field Label
+
+Lux Form Field Label is gebaseerd op Utrecht Form Label.
```

Comment:
Deze zin mag weg. Het is voor de gebruiker niet van belang welk component er onderwater wordt gebruikt.

Het `CitationDocumentation` component is een ander geval. Daarin geven we aan dat de documentatie van Utrecht afkomt.